backend/src/api/cycle.js

```diff
@@ -0,0 +1,69 @@
+const express = require('express')
+const { Cycle } = require('../models')
+const { directorsOnly, errorWrap } = require('../middleware')
+const router = express.Router()
+
+// Get all cycles (previous and current)
+router.get(
```

Comment:
I think we should probably just put this call behind a `[directorsOnly]` security authorization